schema/addgnoll.sql

```diff
@@ -0,0 +1,3 @@
+insert into logfiles (logfiles_i, descr, server, variant, logurl, localfile, dumpurl, oper, static, tz, fpos, lines)values (2025, 'gnollhack.org', 'hdf', 'gnoll', 'https://www.hardfought.org/xlogfiles/gnollhack/xlogfile', 'hdf.gnoll.log','https://www.hardfought.org/userdata/%X/%x/gnollhack/dumplog/%s.gnoll.txt', 't', 'f', 'UTC', 0, 0);
```

Comment:
please add these to the already existing sources schema instead of adding a new flie

Reply:
and just increase sequence number from the last double or triple-digit value